packages/@okta/vuepress-site/.vuepress/nav/reference.js

```diff
@@ -0,0 +1,60 @@
+// the navigation that is shown across the reference pages
+
+module.exports = [
+    {
+        title: "Overview",
+        link: "/docs/reference/"
+    },
+    {
+        title: "OpenID Connect & OAuth 2.0 API",
+        link: "#"
```

Comment:
Are these placeholders?

Reply:
Yes, need to figure out where they need to go.